lib/zipkin-tracer/config.rb

```diff
@@ -53,9 +53,13 @@ def initialize(app, config_hash)
       # This makes it convertible to Amazon X-Ray trace ID format v1.
       # (See http://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-request-tracing.html)
       @trace_id_128bit = config[:trace_id_128bit].nil? ? DEFAULTS[:trace_id_128bit] : config[:trace_id_128bit]
+      # When set to true, only writes a single b3 header for outbound propagation.
+      @write_b3_single_format =
+        config[:write_b3_single_format].nil? ? DEFAULTS[:write_b3_single_format] : config[:write_b3_single_format]
```

Comment:
Is there a glimpse of a convention around naming configuration options across zipkin libraries?
My reasoning is the name could be better but if the other libraries implementing that header format have gone with this one then it's better to conform.

About the name itself, I think it doesn't convey the idea unless you've learned about the "classic" headers and that new "single B3" header beforehand.
I don't know if appropriate but something like "single_b3_header" would be easier to grok IMHO.